hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt

```diff
@@ -144,6 +145,13 @@ The water mask specifies, pixel-by-pixel, where there is land and where there is
 
 *Note that this file is optional. Select the "Include Water Mask" option in the HyP3 Processing Options to include it in the product package.*
 
+-------------
+## 11. Digital Elevation Model
+
+The digital elevation file contains the terrain height for the scenes being processed. The values are taken from the Copernicus GLO-30 Public DEM.
```

Comment:
We've proposed adding "The height values will differ from the original Copernicus DEM dataset, as a geoid correction has been applied" to the product guide.  I'd lean towards adding that note here, as well.

Reply:
Done